setup.cfg

```diff
@@ -1,3 +1,61 @@
+[metadata]
+name = pip
```

Comment:
Maybe this would even work:
```suggestion
name = pip
version = attr: src.pip.__version__
```
(but probably harmful for backwards compat)

Reply:
Won’t work on Python 2.7, so 🙅‍♂️🤷

Reply:
I'd like to use `setuptools_scm` for version. Should we migrate to it?

Reply:
I'm -1 on using `setuptools_scm`.